main.cpp

```diff
@@ -37,27 +37,166 @@ Purpose:  This project continues developing Project3 and will take you through h
  7) After you finish, click the [run] button.  Clear up any errors or warnings as best you can.
  
  */
-
+#include<iostream>
 /*
  copied UDT 1:
  */
+struct DjMixer
+{
+    float channelselected;
+    int volumeOne;
+    int volumeTwo;
+
+    void outputMusic();
+    void increaseVTwo();
+    void increaseVOne();
```

Comment:
Provide the implementation for this member function outside of the class
- [ ] revise